[2022-01-26 17:55:18,052] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: ingest_data.check manual__2022-01-26T17:55:15.944758+00:00 [queued]>
[2022-01-26 17:55:18,076] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: ingest_data.check manual__2022-01-26T17:55:15.944758+00:00 [queued]>
[2022-01-26 17:55:18,076] {taskinstance.py:1238} INFO - 
--------------------------------------------------------------------------------
[2022-01-26 17:55:18,076] {taskinstance.py:1239} INFO - Starting attempt 1 of 1
[2022-01-26 17:55:18,076] {taskinstance.py:1240} INFO - 
--------------------------------------------------------------------------------
[2022-01-26 17:55:18,106] {taskinstance.py:1259} INFO - Executing <Task(BashOperator): check> on 2022-01-26 17:55:15.944758+00:00
[2022-01-26 17:55:18,113] {standard_task_runner.py:52} INFO - Started process 6387 to run task
[2022-01-26 17:55:18,121] {standard_task_runner.py:76} INFO - Running: ['***', 'tasks', 'run', 'ingest_data', 'check', 'manual__2022-01-26T17:55:15.944758+00:00', '--job-id', '107', '--raw', '--subdir', 'DAGS_FOLDER/dag_01.py', '--cfg-path', '/tmp/tmp91fmwi78', '--error-file', '/tmp/tmpy002cywc']
[2022-01-26 17:55:18,123] {standard_task_runner.py:77} INFO - Job 107: Subtask check
[2022-01-26 17:55:18,245] {logging_mixin.py:109} INFO - Running <TaskInstance: ingest_data.check manual__2022-01-26T17:55:15.944758+00:00 [running]> on host 223810fcea28
[2022-01-26 17:55:18,368] {taskinstance.py:1426} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=ingest_data
AIRFLOW_CTX_TASK_ID=check
AIRFLOW_CTX_EXECUTION_DATE=2022-01-26T17:55:15.944758+00:00
AIRFLOW_CTX_DAG_RUN_ID=manual__2022-01-26T17:55:15.944758+00:00
[2022-01-26 17:55:18,370] {subprocess.py:62} INFO - Tmp dir root location: 
 /tmp
[2022-01-26 17:55:18,371] {subprocess.py:74} INFO - Running command: ['bash', '-c', 'echo -------------------------- EMPIEZA ------------------------']
[2022-01-26 17:55:18,387] {subprocess.py:85} INFO - Output:
[2022-01-26 17:55:18,391] {subprocess.py:89} INFO - -------------------------- EMPIEZA ------------------------
[2022-01-26 17:55:18,392] {subprocess.py:93} INFO - Command exited with return code 0
[2022-01-26 17:55:18,437] {taskinstance.py:1277} INFO - Marking task as SUCCESS. dag_id=ingest_data, task_id=check, execution_date=20220126T175515, start_date=20220126T175518, end_date=20220126T175518
[2022-01-26 17:55:18,497] {local_task_job.py:154} INFO - Task exited with return code 0
[2022-01-26 17:55:18,559] {local_task_job.py:264} INFO - 1 downstream tasks scheduled from follow-on schedule check
